tests/components/vicare/test_utils.py

```diff
@@ -0,0 +1,23 @@
+"""Test ViCare utils."""
```

Comment:
When vicare gets more tests I think it makes sense to test this more indirectly instead, as we now test this, but we should also test that this is used

Reply:
Oh nvm, vicare has tests, so I think this would be a nice to have